### DISCUSSION TITLE
Error 500 when trying to join Github Education

### DISCUSSION THREAD

**Discussion by ghost** [community/community#133480] [GitHub Education]
### Hi everyone,

Hello, 

I`m trying to apply myself for the Github Education program. But when I click on "I'm student" button I got this error:

![image](https://github.com/user-attachments/assets/aa8a7706-7305-4ff4-b983-0266f623472e)

Could you help me, please? Thanks.

**Comment by danialtavakoli** [comment]
The 500 error indicates a server issue on GitHub's end. To resolve this, you can try the following steps:

1. **Refresh the Page**: Sometimes, the issue is temporary.
2. **Clear Browser Cache**: This can fix loading problems.
3. **Try a Different Browser**: The issue might be browser-specific.
4. **Check GitHub Status**: Visit [GitHub Status](https://www.githubstatus.com/) to see if there are ongoing issues.
5. **Contact GitHub Support**: If the issue persists, report it to GitHub Support for assistance.

**Reply by krombopulos-eddie** [reply]
Tried your suggestions and the issue persists. No ongoing issues on GitHub Status page either. Will reach out to Support. Thank you.

**Reply by danialtavakoli** [reply]
You're welcome! If you've already tried the suggested steps and the issue is still unresolved, reaching out to GitHub Support is definitely a good next step. They can provide more detailed insights into the runner environment and help troubleshoot any specific issues related to the disk space.

If you have any more questions or need further assistance in the future, feel free to ask. Good luck with resolving the issue, and I hope you get it sorted out soon!

**Comment by george-ckito** [comment]
Still not fixed :(

**Reply by Dawn77** [reply]
So do I :|

**Reply by george-ckito** [reply]
I think it's fixed

**Comment by ravilovrm** [comment]
they fixed
